scipy/stats/_continuous_distns.py

```diff
@@ -5289,7 +5289,8 @@ class kappa4_gen(rv_continuous):
 
     """
     def _argcheck(self, h, k):
-        return h == h
+        shape = np.broadcast_shapes(h.shape, k.shape)
```

Comment:
```suggestion
        shape = np.broadcast_arrays(h, k)[0].shape
```
Again, we have to support 1.19 for now. I was just explaining what will be even better in the future.